pkg/unvanquished_src.dpkdir/bots/default_aliens.bt

```diff
@@ -1,5 +1,48 @@
 selector
 {
+	sequence
+	{
+		condition levelTime >= 60000
+		condition numUsersInTeam == 0 || numOurSpawns < 2
```

Comment:
What's the story behind the num spawns < 2 exception?

Reply:
If all users are in the spawn queue and you have 0 or 1 spanws left, the bots will build new spawns. Even if there is are users on their team. Incidently, this forces a building user to leave 2 spawns at all times, which is universally accepted to be correct.